README.md

```diff
@@ -2,6 +2,28 @@
 
 [![DOI](https://zenodo.org/badge/265256468.svg)](https://zenodo.org/badge/latestdoi/265256468)
 
+## Table of Contents
```

Comment:
Nice to have a clickable table of contents. I found [this online tool](https://ecotrust-canada.github.io/markdown-toc/) that autogenerates a TOC from markdown. You still need to tweak some things in the resultant table, but it does most of the work.